apps/organizations/models.py

```diff
@@ -13,19 +12,18 @@
 ORGANIZATIONS_PAGE_REVERSE_ID = "organizations"
 
 
-class Organization(models.Model):
+class Organization(PageExtension):
     """
-    The Organization model is used to represent and record entities that manage courses.
+    The organization extension represents and records entities that manage courses.
     It could be a university or a training company for example.
 
     This model should be used to record structured data about the organization whereas the
     associated page object is where we record the less structured information to display on the
     page to present the organization.
     """
-    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
Since we are now using DjangoCMS page extensions, I'm not sure we should override it to use UUID. WDYT?